src/cpp/src/sampling/sampler.hpp

```diff
@@ -99,6 +99,7 @@ class Sampler {
     Tokenizer m_tokenizer;
 
     ThreadPool m_thread_pool;
+    std::shared_ptr<ov::op::v0::Constant> m_d2t; // Tensor to store draft2target mapping for eagle model
```

Comment:
Include that comment into var name. That way you make it clear not only here but everywhere the var is ues